### PR TITLE
Reworking Introduction

### PR DESCRIPTION
**Pull request by mattlockyer**
Making language more inclusive and explaining knowledge requirements in detail as opposed to joking about "scary source code".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,14 +14,9 @@ HOW TO USE IT
...
-If you have no idea what I mean by that, or you just clicked that link above and got
-very confused by all these strange characters that got displayed on your screen,
-it's a sign, that impress.js is not for you.
+impress.js was created with developers in mind and some basic knowledge of html, css and javascript is a prerequisite to install and utilize it's features.
 
-Sorry.
-
-Fortunately there are some guys on GitHub that got quite excited with the idea of building
-editing tool for impress.js. Let's hope they will manage to do it.
+If you are a designer or novice developer and intent on using impress, there is a project on GitHub that aims to provide an editing tool for impress.js. It is still in development but we hope they will manage to make impress.js more accessible to all.
```
Comment (FagnerMartinsBrack) [inline README.md]: Unfortunately this also changed the intent of the sentence. It is not just one project that aims to provide an editing tools, there are [more than one of them](https://github.com/impress/impress.js/wiki/Examples-and-demos/5d887507caa5cf534eab6713d9adb3a5e7662459). Can we focus in the first paragraph only though? Or this paragraph also causes trouble?

File: README.md [hunk 2]
```diff
@@ -88,7 +83,7 @@ For browsers not supporting CSS3 3D transforms impress.js adds `impress-not-supp
 class on `#impress` element, so fallback styles can be applied to make all the content accessible.
 
 
-### Even more explanation and technical stuff
+### Even more explanation and technical stuff (and some opinions)
```
Comment (FagnerMartinsBrack) [inline README.md]: Why opinions? The text is pretty objective in what one is supposed to expect from impress.js in its current state.

**Comment by FagnerMartinsBrack** [comment]
@mattlockyer, still tracking this?

**Comment by mattlockyer** [comment]
Not particularly sorry. I have offered my suggestions and now believe that
the Impress mods are capable of a proper rewrite.

Good luck!

**Comment by FagnerMartinsBrack** [comment]
Closing due to the lack of feedback from the [OP](https://medium.com/@fagnerbrack/what-does-op-means-on-github-8db958b604af).

I have create a new PR for this, see #562. @mattlockyer can you review that PR if possible?

**Pull request closed without merging**